util/find-doc-nits

```diff
@@ -864,19 +864,15 @@ if ( $opt_c ) {
 }
 
 if ( $opt_l ) {
-    foreach (@ARGV ? @ARGV : (glob('doc/*/*.pod'), glob('doc/*/*.pod.in'),
-                              glob('doc/internal/*/*.pod'))) {
+    foreach ( glob('doc/*/*.pod doc/internal/*/*.pod') ) {
```

Comment:
Why do we no longer check ARGV?

Reply:
Because when checking links, you have to look at all the other manpages to see if there's content so that the links are valid; it makes no sense for -l.